frontend/.gitignore

```diff
@@ -26,19 +26,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-package-lock.json
-
-# compiled patternfly assets
-src/**/*.css
-src/**/*.css.map
-!src/index.css
-!src/css/.gitkeep
-src/img/*
-!src/img/.gitkeep
-src/fonts/*
-!src/fonts/.gitkeep
-src/sass/*
-!src/sass/index.sass
-
-# Commit Slider css
-!src/components/IstioWizards/Slider/styles/default.css
+package-lock.json
```

Comment:
Why `package-lock.json` is ignored? [Not a good idea.](https://dev.to/adamklein/package-lock-json-in-git-or-not-50l5)

Reply:
answered here https://github.com/kiali/kiali/pull/6426#discussion_r1279663323